packages/web-twig/src/Resources/components/Breadcrumbs/index.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set _class = props.class | default(null) -%}
+{%- set _elementType = props.elementType | default('nav') -%}
+{%- set _goBackItemTranslation = props.goBackItemTranslation | default('Back') -%}
+{%- set _items = props.items | default([]) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Breadcrumbs' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<{{ _elementType }} {{ mainProps(props) }} {{ classProp(_classNames) }} aria-label="Breadcrumb">
+{% apply spaceless %}
+    {%- if items|length > 0 -%}
```

Comment:
```suggestion
    {%- if _items|length > 0 -%}
```

Reply:
That's what was causing the "weird" behavior. Thank you 👍